config/config.go

```diff
@@ -106,6 +106,7 @@ type Configuration struct {
 	WfStepGracefulTerminationTimeout time.Duration `yaml:"wf_step_graceful_termination_timeout,omitempty" mapstructure:"wf_step_graceful_termination_timeout"`
 	PurgedDeploymentsEvictionTimeout time.Duration `yaml:"purged_deployments_eviction_timeout,omitempty" mapstructure:"purged_deployments_eviction_timeout"`
 	ServerID                         string        `yaml:"server_id,omitempty" mapstructure:"server_id"`
+	ClusterID                        string        `yaml:"cluster_id,omitempty" mapstructure:"cluster_id"`
```

Comment:
You can bind this field with env var and command option as done by instance with ServerID.
Need to update documentation about it too.

Reply:
To me a Yorc cluster should be bind to a Consul cluster. 
So why not use the Consul DC name here?

Reply:
Sounds good if we are sure a it's unique per Yorc cluster, no need to add ClusterID. 
But it doesn't seem to be required and maybe Yorc use a default value when not. Is there a way for me to get this value ?

Reply:
Ok I've checked , it's set to dc1 if not defined in config. I will use this rather than defining a new config entry !